.github/workflows/pr-cloud-run.yaml

```
@@ -21,16 +50,27 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@34e114876b0b11c390a56381ad16ebd13914f8d5 # v4
         with:
           submodules: false
 
       # Setup gcloud CLI
       - id: "auth"
-        uses: "google-github-actions/auth@v1"
+        if: ${{ secrets.STAGE_GCP_WIF_PROVIDER != '' && secrets.STAGE_GCP_WIF_SERVICE_ACCOUNT != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)